pact/ledger.pact

```diff
@@ -110,6 +110,12 @@
     (compose-capability (UPDATE_SUPPLY))
   )
 
+  (defconst MARMALADE_GUARD "marmalade-guard")
+
+  (defun ledger-guard:guard ()
```

Comment:
Let's go ahead and move `enforce-ledger` in here so that we don't have to define it in policies. We can even add it to `poly-fungible-v2`: 
```
(defun enforce-ledger ()
  @doc "Enforce ledger module guard for policies to be able to validate access to policy operations."
  (enforce-guard (create-module-guard "ledger-guard"))
)
```
Then you can just directly call this in module ops.

Reply:
As reported by @ggobugi27 this doesn't work, retracting